### PR TITLE
crud shift

### PR DESCRIPTION
**Pull request by yarinTag**

### CODE REVIEW

File: src/modules/shifts/dto/CreateRequest.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { IsBoolean, IsUUID } from 'class-validator';
+import { LocalDateTime } from '@js-joda/core';
+
+export class CreateRequest {
+  @IsBoolean()
+  active: boolean;
+
```
Comment (yariv245) [inline src/modules/shifts/dto/CreateRequest.ts:7]:
```suggestion
@IsNotEmptyObject()
```

File: src/modules/shifts/dto/CreateRequest.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  start: LocalDateTime;
+
```
Comment (yariv245) [inline src/modules/shifts/dto/CreateRequest.ts:10]:
```suggestion
@IsNotEmptyObject()
```

File: src/modules/shifts/dto/CreateRequest.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { IsBoolean, IsUUID } from 'class-validator';
```
Comment (yariv245) [inline src/modules/shifts/dto/CreateRequest.ts]:
```suggestion
import { IsBoolean,IsNotEmptyObject ,IsUUID } from 'class-validator';
```

File: src/modules/shifts/dto/UpdateRequest.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { LocalDateTime } from '@js-joda/core';
+
+import { IsOptional, IsUUID, IsBoolean } from 'class-validator';
+
+export class UpdateRequest {
+  @IsUUID()
+  id: string;
+
+  @IsOptional()
```
Comment (yariv245) [inline src/modules/shifts/dto/UpdateRequest.ts:9]: add the @IsNotEmptyObject() like in create request

**Review by yariv245 — APPROVED** [review]

File: src/modules/shifts/dto/UpdateRequest.ts (hunk 4 above)
Comment (yarinTag) [reply src/modules/shifts/dto/UpdateRequest.ts:9]: It's not relevant bc we already have the @IsOptional()

**Pull request merged**